ui/src/main/java/com/hyperwallet/android/ui/transfermethod/SelectTransferMethodPresenter.java

```diff
@@ -35,14 +39,17 @@
 public class SelectTransferMethodPresenter implements SelectTransferMethodContract.Presenter {
     private static final String TAG = SelectTransferMethodPresenter.class.getName();
 
-    private static final String INDIVIDUAL = "INDIVIDUAL";
     private final TransferMethodConfigurationRepository mTransferMethodConfigurationRepository;
+    private final UserRepository mUserRepository;
     private final SelectTransferMethodContract.View mView;
+    private HyperwalletUser mUser;
```

Comment:
We don't need to keep a reference to the HyperwalletUser - we're already doing it in the repository. With that said, please revert all those "build" callback methods. Most of that refactoring is actually not necessary as we only need the profile type (nesting the get user call) in some specific cases

Reply:
@fmattos-hw Done